src/main.js

```diff
@@ -33,9 +25,18 @@ let load = base.loader({
     }
   },
 
+  monitor: {
+    requires: ['process', 'profile', 'cfg'],
+    setup: ({process, profile, cfg}) => base.monitor({
+      project: cfg.monitor.component,
```

Comment:
I think it would be the `process` option in https://github.com/taskcluster/taskcluster-lib-monitor#options-and-defaults

Perhaps this should be in the example at the top since we'll almost always want it?